src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,210 @@
+        }
+        databaseCleanup.execute();
+        initStations();
+    }
+
+    private void initStations(){
+        stationRepository.save(new Station("우장산역"));
+        stationRepository.save(new Station("화곡역"));
+        stationRepository.save(new Station("까치산역"));
+        stationRepository.save(new Station("마곡나루역"));
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+        // when
+        String location = insertLineSuccess(line_1).header("Location");
+
+        // then
+        String lineName = selectLine(null, location).extract().jsonPath().get("name");
+        assertThat(lineName).isEqualTo(line_1.getName());
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철노선 목록을 조회한다.")
+    @Test
+    void getLines() {
+        // given
+        insertLineSuccess(line_1);
+        insertLineSuccess(line_2);
+
+        // when
+        List<String> lineNames =
+                selectLines().extract().jsonPath().getList("name", String.class);
+
+        // then
+        assertAll (
+                () -> assertThat(lineNames.size()).isEqualTo(2),
+                () -> assertThat(lineNames).contains(line_1.getName(), line_2.getName())
+        );
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void getLine() {
+        // given
+        String location = insertLineSuccess(line_1).header("Location");
+
+        // when
+        String lineName = selectLine(null, location).extract().jsonPath().get("name");
+
+        // then
+        assertThat(lineName).isNotNull();
+        assertThat(lineName).isEqualTo(line_1.getName());
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 수정하면
+     * Then 해당 지하철 노선 정보는 수정된다
+     */
+    @DisplayName("지하철 노선을 수정한다.")
+    @Test
+    void updateLine() {
+        // given
+        String location = insertLineSuccess(line_1).header("Location");
+
+        // when
+        updateLineSuccess(null, location, line_2);
+
+        // then
+        String lineName = selectLine(null, location).extract().jsonPath().get("name");
+        assertThat(lineName).isEqualTo(line_2.getName());
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 삭제하면
+     * Then 해당 지하철 노선 정보는 삭제된다
+     */
+    @DisplayName("지하철역을 삭제한다.")
+    @Test
+    void deleteLine() {
+        // given
+        String location = insertLineSuccess(line_1).header("Location");
+
+        // when
+        deleteLineSuccess(null, location);
+
+        // then
+        String lineName = selectLine(null, location).extract().jsonPath().get("name");
+        assertThat(lineName).isNullOrEmpty();
+    }
+
+    private ValidatableResponse selectLine(Long id, String location) {
```

Comment:
```suggestion
    private ValidatableResponse selectLine(String location) {
```
모든 테스트에서 id 파라미터에 null을 넣어주고있네요.
id를 받지 않아도 될거같아요!